.gitlab/kernel_version_testing/system_probe.yml

```diff
@@ -403,7 +404,19 @@ kernel_matrix_testing_run_tests_x64:
     matrix:
       - TAG: ["ubuntu_16.04", "ubuntu_18.04", "ubuntu_20.04", "ubuntu_22.04", "ubuntu_23.10", "amzn_4.14", "amzn_5.4", "amzn_5.10", "fedora_37", "fedora_38", "debian_10", "debian_11", "debian_12", "centos_79"]
 
-kernel_matrix_testing_run_tests_arm64:
+kmt_run_tests_no_tracer_x64:
+  extends:
+    - .kmt_run_tests_x64
+  variables:
+    TEST_SET: "no_tracer"
```

Comment:
I think this could be another variable in the `matrix` in `kernel_matrix_testing_run_tests_*` rather than defining two new jobs. 
`- TEST_SET: ["no_tracer", "only_tracer"]`

Reply:
IIRC I was hitting a gitlab limit on the number of child jobs for the matrix `TAG X TEST_SET`.
I will try again to be sure.

Reply:
Please use a different name, `tracer/no tracer` is not a good cut between USM and NPM tests

Reply:
This is not a separation of USM and NPM tests. It is a separation of `TestUSMSuite` and `TestTracerSuite` since these are the two longest running suites.

Reply:
> The number of permutations cannot exceed 200
It seems we should be ok on the Gitlab limit front.

Reply:
`TestUSMSuite` runs a tracer as well so the split `tracer/no tracer` is not a good split

Reply:
@guyarb are you objecting to the _naming_ of the split, or the split itself? The split is purely based on runtime, since the optimal way is to divide into two roughly equivalent sets.

Reply:
Only to the naming
I'm supportive of the change in general as long as it works without any impact

Reply:
👍 We will come up with a better name